core/src/main/java/kafka/log/remote/RemoteLogManager.java

```diff
@@ -212,9 +213,9 @@ public RemoteLogManager(RemoteLogManagerConfig rlmConfig,
         rlmCopyQuotaManager = createRLMCopyQuotaManager();
         rlmFetchQuotaManager = createRLMFetchQuotaManager();
 
-        indexCache = new RemoteIndexCache(rlmConfig.remoteLogIndexFileCacheTotalSizeBytes(), remoteLogStorageManager, logDir);
```

Comment:
Part of the value of AbstractConfig subclasses is providing convenient and well-typed functions instead of requiring other classes to depend on the _PROP constants.

Reply:
@gharris1727 do you have any specific reference here, if so, pls share ? I tried to see in a few but couldn't.

Reply:
See here: https://github.com/apache/kafka/blob/8b3c77c6711d8c1424659a1b4621a8c823fa2236/connect/runtime/src/main/java/org/apache/kafka/connect/runtime/ConnectorConfig.java#L269-L301

Reply:
Indeed. Its much smaller. thanks. Updated.